include/EffectBase.h

```diff
@@ -31,6 +31,8 @@
 #ifndef OPENSHOT_EFFECT_BASE_H
 #define OPENSHOT_EFFECT_BASE_H
 
+#define OPENSHOT_PLUGIN_API_VERSION 0x0001
```

Comment:
Yeah... as you mentioned in the main PR comments, I'd just use the SO version instead. (It's `#define`d as `OPENSHOT_VERSION_SO` in `OpenShotVersion.h`.)

The main reason being, we really don't _have_ an **effects** API currently (and that's on us). The abstractions for effects use aren't really great. Callers generally need to have knowledge of the API for the specific effects they use, and to an extent that will also include the dynamic ones. (The JSON interfaces do _somewhat_ alleviate that, but more on that later.)

As a result, the plugin compatibility will most likely be tied to the library API **even if** the plugin-_loading_ API stays the same, and a plugin API versioning scheme just adds another numbering to keep track of.

But if this **IS** retained, it should be placed in `include/OpenShotVersion.h.in` instead of here. (Which is the input for a CMake-generated header file that you can include via `#include "OpenShotVersion.h"`, to access any of the versioning macros.)